server/protocols/webadmin-cli/src/main/java/org/apache/james/cli/user/UserCommand.java

```diff
@@ -59,7 +60,7 @@ public Integer call() {
     }
 
     public Feign.Builder feignClientFactory() {
-        return new FeignClientFactory(new JwtToken(webAdminCli.jwt)).builder()
+        return new FeignClientFactory(new JwtToken(Optional.ofNullable(webAdminCli.jwt))).builder()
```

Comment:
This line is duplicated between UserCommand and DomainCommand.

Can we move it to WebAdminCli?